framework/source/class/qx/ui/window/Window.js

```diff
@@ -929,6 +1055,40 @@ qx.Class.define("qx.ui.window.Window",
       }
     },
 
+    // overridden
```

Comment:
Afaik applyers are not created automatically and then overriden. If you declare a property and define there an applyer attribute, then using that property via the setter will throw an exception. See http://tinyurl.com/zsd4oaq